tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/api/ProgrammeMembershipResourceIntTest.java

```diff
@@ -313,24 +314,24 @@ void shouldGetProgrammeMembershipSummaryListByUuid2() throws Exception {
     curriculumMembership.setCurriculumEndDate(today.plusDays(1));
     programmeMembershipRepository.saveAndFlush(programmeMembership);
 
-    UUID savedProgrammeMembershipUuid = programmeMembership.getUuid();
+    String ProgrammeMembershipUuid = String.valueOf(programmeMembership.getUuid()) + ",77777";
+    String FilterdUuidList = String.valueOf(programmeMembership.getUuid());
```

Comment:
Variable names should be lower camel case, can also just use `programmeMembership.getUuid().toString()` instead of `String.valueOf()` since we know it's not going to be null.